<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMapDialog</class>
 <widget class="QDialog" name="NewMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Map</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QLabel" name="lblMapName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>This is the name of the map which will be visible to players while they are on it.</string>
    </property>
    <property name="text">
     <string>Map Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMapName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>26</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblWidth">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>This is the width of the map, in tiles.</string>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnWidth">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>52</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>16</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnHeight">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>82</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>16</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblHeight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>89</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>This is the height of the map, in tiles.</string>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
